### PR TITLE
Polysmorph tweaks

### PR DESCRIPTION
**Pull request by Moltijoe**
Polysmorph have a lacklustre identity, they're pretty much just reskinned lizards with acid blood
These are just a few tweaks to try and give them a bit of a niche following the idea of "they're basically just xeno human hybrid experiments"
So they're going to be more physically capable, but worse at doing regular tasks
since these are all just stat tweaks, it won't change the regular gameplay loop much, but there's not any other ideas i have currently.

:cl:  
rscdel: polysmorph no longer have bones (this is a good thing)
tweak: polysmorph do tasks slower
tweak: polysmorph move slightly faster
tweak: polysmorph have less innate wound armour
tweak: polysmorph have 0.9 brutemod from 1
tweak: polysmorph have 1.35 burnmod from 1.25
tweak: polysmorph have a high-end punch damage of 11 from 10
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
Making them faster might be a bad idea, since it makes any that aren't wearing clothing with slowdown much more robust in melee combat and much more able to disengage when a fight starts going bad.

**Comment by Moltijoe** [comment]
> Making them faster might be a bad idea, since it makes any that aren't wearing clothing with slowdown much more robust in melee combat and much more able to disengage when a fight starts going bad.

ethereals already move faster, no one notices

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 	taste_description = "slime"
 
 /datum/reagent/vaccine/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
-	if(islist(data) && (method == INGEST || method == INJECT))
+	if(islist(data) && method == INJECT)
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: ????

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Moltijoe) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: save me from this torment, what did you do to the main branch when you merged that PR?

**Comment by 00ze-cyclone** [comment]
Polysmorphs have limited night vision, their special liver, the ability to breathe both plasma and oxygen and they take less damages from acid. They are already unique enough and simple stat changes won’t change how they feel anyway.

**Comment by MajManatee** [comment]
Pretty sure skrem is planning on making changes to all races in the near future, so this feels unnecessary

**Comment by Moltijoe** [comment]
> Polysmorphs have limited night vision, their special liver, the ability to breathe both plasma and oxygen and they take less damages from acid. They are already unique enough and simple stat changes won’t change how they feel anyway.

preterni also have night vision and sort of a "special liver"
acid damage isn't really something that takes effect all that often
honestly though, i'm not feeling these changes quite as much as the lizard ones, 
i'll wait and see what the balance council thinks

**Comment by 00ze-cyclone** [comment]
I mean balance wise it's probably fine, it's just a lot of small stat changes that don't really make the specie feel any different.
Lizard are more mechanics revolving around temperature, polys aren't getting any here so it just feel inconsequential.

**Comment by Moltijoe** [comment]
> I mean balance wise it's probably fine, it's just a lot of small stat changes that don't really make the specie feel any different. Lizard are more mechanics revolving around temperature, polys aren't getting any here so it just feel inconsequential.

yeah, if i end up having any other ideas for this species, i'll try to add them, but it's hard when the direction they'd go brings them dangerously close to what is ideal for validhunters or antags.

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**